addons/hr_recruitment_survey/security/hr_recruitment_survey_security.xml

```diff
@@ -5,9 +5,19 @@
             <field name="implied_ids" eval="[Command.link(ref('survey.group_survey_user'))]"/>
             <field name="comment">It will also allow to send surveys and see the resume.</field>
         </record>
-        <record id="survey.survey_user_input_rule_survey_user_read" model="ir.rule">
-            <field name="name">Survey user input: officer: read all non private survey answers</field>
-            <field name="domain_force">[('applicant_id', '=', False)]</field>
+        <record id="survey_user_input_rule_recruitment_survey_user" model="ir.rule">
+            <field name="name">Survey user input: officer: read unrestricted applicant answers or in restricted users</field>
+            <field name="model_id" ref="survey.model_survey_user_input"/>
+            <field name="domain_force">[
+                ('applicant_id', '!=', False),
```

Comment:
It should be `('applicant_id', '=', False)`, but even then, it won't do a trick because group-specific rules are combined with OR and this won't prevent survey user from reading recruitment answers

So, if we don't want to overwrite survey rules in hr_recruitment_survey and still not let survey users read recruitment answers, I don't see other way but to add another `survey_type` for recruitment. I will try to do that and let's see what comes out of it :crossed_fingers: